docs/architecture/runtime-agent-contract.md

```diff
@@ -0,0 +1,27 @@
+# Runtime Agent contract
+
+## Overview
+
+This document describes contract between [Management Plane](../terminology.md#management-plane) and [Runtime Agent](../terminology.md#runtime-agent). Main responsibilities of the Runtime Agent are:
+- [Establishing trusted connection](#establishing-trusted-connection)
+- [Renewing trusted connection](#renewing-trusted-connection)
+- [Configuring applications](#configuring-applications)
```

Comment:
The link should look like this to work correctly:
`- [Configuring the runtime](#configuring-the-runtime)`